packages/client/src/views/VerifyCertificate/VerifyCertificatePage.tsx

```diff
@@ -508,24 +507,24 @@ export function VerifyCertificatePage() {
                       }
                       value={
                         <Text variant={'reg16'} element={'span'}>
-                          {getChildOrDeceasedData(currentData).fullName}
+                          {getFullName(data)}
```

Comment:
this "getXyz" paradigm isn't necessary We should rather early-return the loading state if data is not loaded, and then directly render the stuff we want without single-use helper functions. This component would be simpler that way. I can look into refactoring this.

Reply:
yeah we could even use `isBirthRegistration` early on recordDetails to render entirely different component like `<BirthVerifyDetails>` & `<DeathVerifyDetails>`

Reply:
Let's leave the refactoring for another PR if we wanna do it. Let's fix the bug here